src/cljs/redditv/reddit.cljs

```diff
@@ -4,7 +4,7 @@
             [redditv.jsonp :refer [send-jsonp]]
             [redditv.youtube :as yt]))
 
-(def reddit-url "http://www.reddit.com")
+(def reddit-url "https://www.reddit.com")
```

Comment:
Thanks for the contribution @fromheten! I'll take a look at this after work :+1: